packages/forms-web-app/src/lib/multifile-upload.js

```diff
@@ -0,0 +1,27 @@
+const getValidFiles = (errors, files) => {
```

Comment:
I think this file could potentially have a more specific name to better describe its purpose

Reply:
changed to `multi-file-upload-helpers.js`